prov/efa/src/rxr/rxr.h

```diff
@@ -165,6 +165,7 @@ static inline void rxr_poison_mem_region(uint32_t *ptr, size_t size)
  */
 #define RXR_NO_COMPLETION	BIT_ULL(60)
 #define RXR_NO_COUNTER		BIT_ULL(61)
+#define RXR_DELIVERY_COMPLETE_REQUESTED		BIT_ULL(62)
```

Comment:
Can we not put this in ```rx_entry->fi_flags```? I think ```rx_entry``` has flags? If not, a separate variable is fine. there is only 2 provider specific flags left, and I do not want to use it here.

Reply:
By saying "separate variable", are you talking about adding another member in `struct rxr_rx_entry` ?

Reply:
For rx_entry we can use ```rx_entry->rxr_flags```. For tx_entry, I think either add ```rxr_flags``` or a separate member.

Reply:
`tx_entry->rxr_flags` is added, resolving.